README.md

```diff
@@ -102,32 +99,28 @@ BIC = n*log(residual sum of squares/n) + K*log(n)
 ```
 
 where:
+
 - n: number of observations
 - K: number of parameters (including intercept)
 
 #### Function
 
 ```
-bic(x, y, n, k, model = 'linear')
+bic(y, y_pred, p)
 ```
 **Parameters:**
-* **x**: array (n_samples, n_features)
-  * Predictive variable(s)
-
-* **y**: array (n_samples, n_targets)
-  * Target variable(s)
+* **y**: array-like of shape = (n_samples) or (n_samples, n_outputs)
+  * True target variable(s)
 
-* **n**: int
-  * Number of observations
+* **y_pred**: array-like of shape = (n_samples) or (n_samples, n_outputs)
+  * Fitted target variable(s) obtained from your regression model
 
-* **k**: int
+* **p**: int
   * Number of predictive variable(s) used in the model
 
-* **model**: default 'linear' | 'logistic' | 'ridge' | 'lasso' | 'elasticnet'
-  * Method applied to the model
-
 **Return:**
-* BIC score of the model: int
+* bic_score: int
+  * BIC score of the model
 
 ### Mallow's C_p
 
```

Comment:
Can you add math function of mallow?

Reply:
I already have math function like `C_p = (SSE_p/MSE) - (n - 2p)`. Or you mean something else?

Reply:
Yea that's the function. I don't see it in your pull request though

Reply:
We already included it in current version. I did not delete it. You can see it in current repo.